sass/layout/_markdown.scss

```diff
@@ -330,12 +330,11 @@ h6.markdown__heading {
 
 .post__body {
     hr {
-        height: 4px;
+        height: 1px;
         padding: 0;
         border: 0 none;
         margin: 15px 0 16px;
-        background-color: #e7e7e7;
-        opacity: 0.2;
+        background-color: rgba(var(--center-channel-text), 0.2) !important;
```

Comment:
@sinansonmez is it necessary to use Important here, Would be great if we could avoid by using some class combination with more specificity.

Reply:
I also didn't feel comfortable to put `!important`. That is why also I re-requested review from you 😄 
However current `background-color` is coming from `inline` styling most probably from the line below. 

I will do a deep dive to do it without `!important`

https://github.com/mattermost/mattermost-webapp/blob/2719a868113c3db10f2beb7b701552f7e7043a8d/utils/utils.tsx#L423

Reply:
also a quick side note, im not sure why but its rendering out as (0,0,0,0) for some reason... but I agree we should be avoiding the use of an !important tag
<img width="552" alt="image" src="https://user-images.githubusercontent.com/31866506/189382109-a236b591-eb3a-45b4-a4a8-7145c6ba4918.png">

(And I don't mean to be fussy the opacity should be .24)

Reply:
Now it should be fixed without using `important`